lib/runner/cli/cli.js

```diff
@@ -337,6 +337,12 @@ class CliRunner {
     return this;
   }
 
+  get isMobile() {
+    const {platformName} = this.test_settings.desiredCapabilities || {};
+
+    return platformName ? (platformName.toLowerCase() === 'ios' || platformName.toLowerCase() === 'android') : false;
+  }
```

Comment:
I think this helper is already present in Nightwatch

Reply:
Yes but that particular helper will execute after the requirement. And it's not a good idea to create an instance of `NightwatchClient` first to access `api` which contains `isMobile` helper.